### DISCUSSION TITLE
Week 12 - Getting Prepared for Final Submission and checklist completion

### DISCUSSION THREAD

**Discussion by bst3ck** [USU-ITLS-x265-23-summer/class-repo#235] [General]
## Week 12 Progress Update
So I may have under estimated the amount of work that hooking up the javascript would be. In my original project plan, I had anticipated the front end taking the longest and hooking up the javascripts to the API service the easier of the tasks. Well, that did not work out as I had planned. Each time I conducted my end to end test, I found another bug or an issues with one of the API calls. I had attempted to re-use some of the patterned items I had on various pages, but I ended up coding up the script anyway since either the data fields were different or the layout needed tweaking etc. What I ended up was not where I had hoped to be, but it is close. I had intended to spend more time working on some tutorials and embedded videos. But at this point, I am just happy it is functioning.

Here is a link to the site with the latest code:
[http://www.bksteckler.com/klokke/](http://www.bksteckler.com/klokke/)

#### Finalization Tasks
 - Review ADA requirements to ensure all critical errors are resolved
 - Conduct end to end test
 - Review usability issue and build our phase 2 backlog

#### Build Page Framework
 - ~~Global Header and Footers~~
 - ~~Global CSS Stylesheet~~

#### Build Homepage (HTML)
 - ~~Layout Goal Section~~
 - ~~Daily Tasks Action List~~
 - ~~Client Summary~~

#### Build Time Tracking Page
 - ~~Activity Tracker (HTML & CSS)~~
 - ~~Build Javascript for timer and data storage~~

#### Build Goal Tracker Page
 - ~~Build Goal setting layout~~
 - ~~Goal table along~~
 - ~~Add new goal submission forms~~

#### Build a Client Management page
 - ~~Basic layout~~
 - ~~Current Year Summary Table~~
 - ~~Current project list~~
 - ~~Add new project / milestone / task~~

**Comment by CJ3315** [comment]
You have made some great progress the cite looks great!

**Comment by audreyhki** [comment]
Nice job! I like the color gradient on the form that is provided to fill out.

**Comment by kassyoveson** [comment]
Great job on your final project. I am curious about a few things. Why did you quote Steve Jobs at the end? I like the quote, I as curious as to why you chose it. I like the theme of the work life balance. I also liked your color choices, and the bar graphs look really nice. I am sure those were not easy to create.

**Comment by beto0890** [comment]
Looks great, and you have made significant progress.  Great design, it really gets your attention.

**Comment by gabebott** [comment]
Wow! I am super impressed with this! I think you did a great job on this site. I can tell how much time you must have put into this! Great job!
